include/wx/private/display.h

```diff
@@ -144,14 +126,22 @@ class wxDisplayImpl
     virtual bool ChangeMode(const wxVideoMode& mode) = 0;
 #endif // wxUSE_DISPLAY
 
+    // return true, if this display is still connected physically to system
+    virtual bool IsConnected() const { return m_isConnected; }
+
+    // indicate that this display is not connected to system anymore
+    virtual void Disconnect() { m_isConnected = false; }
+
```

Comment:
Minor, but it doesn't seem like either of these functions needs to be `virtual`.

As it's a private class, we can always add `virtual` later if necessary, but it's better not to add it without need, so I'd just remove it:

```suggestion
    // return true, if this display is still connected physically to system
    bool IsConnected() const { return m_isConnected; }

    // indicate that this display is not connected to system anymore
    void Disconnect() { m_isConnected = false; }

```